web-ui/src/main/resources/catalog/templates/top-toolbar.html

```diff
@@ -140,7 +140,7 @@
 
     <form class="navbar-form navbar-right language-switcher">
       <span class="gn-menuheader-xs visible-xs"
-            data-ng-if="!authenticated && service !== 'catalog.signin' && service !== 'new.account' && !shibbolethEnabled"
+            data-ng-if="!authenticated && service !== 'catalog.signin' && service !== 'new.account' && !shibbolethHideLogin"
```

Comment:
Same as previous, would not be better to check both `shibbolethEnabled` and `shibbolethHideLogin`?